dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -37,6 +37,7 @@ autorestart=false
 stdout_logfile=syslog
 stderr_logfile=syslog
 
+{% if skip_ledd != 1 %}
```

Comment:
Debating whether these values should be `True` instead of `1` to be more Pythonic. Thoughts?

Reply:
@jleveque I change the values from int to boolean. 
the current implementation(already supported in the previous commit) is like this:
 1. set xxx_skip = true in the json, the conf will NOT be generated, 
 2. if xxx_skip not set in the json, the conf will be generated, 
 3. if xxx_skip set to false in the json, the conf will also be generated.
  
I also removed the "xxx_skip = fase" setting from the Mellanox platform configuration to avoid missleading.

Reply:
@jleveque due to some tricky conflicts that I am not able to resolve, I choose to close this PR and opened a new one: https://github.com/Azure/sonic-buildimage/pull/2654 please switch to the new one, sorry for the inconvenience.